robot_control_info: {
  kw: 0.01,
  kv: 1.2,
  d: 0.01,
  kd: 1
}